packages/Algernon.package/ALGCore.class/instance/handleListenEvent..st

```diff
@@ -1,7 +1,13 @@
 events-processing
 handleListenEvent: anEvent
-	| isActivationString shouldToggle |
+	| isActivationString shouldToggle timestamp difference |
+	timestamp := TimeStamp current time asNanoSeconds.
+	difference := timestamp - (lastHalfActivationTimestamp ifNil: [0]).
```

Comment:
Would maybe rename `lastHalfActivationTimestamp`, because it is not necessarily a half activation. Better imo: `lastActivationKeyPressTime`(stamp)
Would've helped my understanding